### PR TITLE
Add lessons fetching

### PR DESCRIPTION
**Pull request by SaraVieira**
- fetches lessons
- fetches single lesson

### CODE REVIEW

File: src/pages/courses/[slug]/[slug_lesson].tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { prisma } from '~/server/prisma';
```
Comment (TejasQ) [inline src/pages/courses/[slug]/[slug_lesson].tsx:1]: Question: why do we expose lessons publicly? Aren’t those for people who only buy the course?

Comment (SaraVieira) [reply src/pages/courses/[slug]/[slug_lesson].tsx:1]: Yes, I have to after lock it but the issue is you can't fetch from FS using server side props

Comment (SaraVieira) [reply src/pages/courses/[slug]/[slug_lesson].tsx:1]: I will always show blank before js loads basically

Comment (TejasQ) [reply src/pages/courses/[slug]/[slug_lesson].tsx:1]: Awesome let's 🚢

**Review by TejasQ — APPROVED** [review]

**Pull request merged**